<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFunctionTest</class>
 <widget class="QWidget" name="CFunctionTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>72</y>
     <width>112</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>72</y>
     <width>112</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>72</y>
     <width>112</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>sure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
